src/blueapi/service/main.py

```diff
@@ -142,6 +158,7 @@ def get_device_by_name(name: str, runner: WorkerDispatcher = Depends(_runner)):
     response_model=TaskResponse,
     status_code=status.HTTP_201_CREATED,
 )
+@start_as_current_span(TRACER, "request", "task.name", "task.params")
```

Comment:
Yes, this is enabled by one of the environment variables

Reply:
Though it now transpire that this can be enabled by passing parameters into the FastAPI Instrumentor constructor so I 've switched it to use that method instead